kong.conf.default

```diff
@@ -1017,14 +1020,16 @@
                                  # for a short period of time after Routes and
                                  # Services updates.
 
-#router_update_frequency = 1     # Defines how often the router changes are
+#worker_state_update_frequency = 5
+                                 # Defines how often the worker state changes are
                                  # checked with a background job. When a change
-                                 # is detected, a new router will be built. By
-                                 # default we check for changes every second.
-                                 # Raising this value will decrease the load on
-                                 # database servers and result in less jitter
-                                 # in proxy latency, with downside of longer
-                                 # converge time for router updates.
+                                 # is detected, a new router or balancer will be
+                                 # built, as needed. By default we check for
+                                 # changes every second. Raising this value will
```

Comment:
The text says "by default we check for changes every second" but the config option itself here says `5` (I also note this was a change from the previous behavior — is this deliberate? Also, `kong_defaults.lua` below still says `1`, so this file is lying right now).

I think the text should not contain this sentence about the default, because the [generated documentation page](https://docs.konghq.com/2.0.x/configuration/#router_update_frequency) builds the "Default:" line out of the entry itself.